.eslintignore

```diff
@@ -1,3 +1,3 @@
-dist/
-nearcore/
-protos.js
+packages/near-api-js/dist/
```

Comment:
I think we can create `.eslintignore` in `packages/near-api-js` to scope it to that directory.

Reply:
Do any of these actually exist though? I couldn't find them. Is this still needed at all?

Reply:
Yeah prob cleaner to scope this to package. Yeah all these are generated after some scripts e.g. browserify run